### ISSUE TITLE
rendering problems on IRIS gpu

### ISSUE THREAD

**Issue by haraldsteinlechner** [pro3d-space/PRo3D#116]
currently there seem to be rendering problems on intel iris hardware. We have a repro on a project member's computer. After writing artificial rendering tests which all worked pro3d has now support to set render control properties such as multisampling count and mapping etc.

The attempt can be found here: https://github.com/pro3d-space/PRo3D/commit/844d257b3a8a17e3b10c4e3168b849c412825c03

**Comment by haraldsteinlechner** [comment]
Thanks to Kens testing sessions we could pin the problem down to a problem with multisampled rendertargets. The current workaround for iris integrated graphics is currently to set samples to 1. 
The problem is still under investigation @hyazinthh  @luitheking since it is not clear whether this is a driver bug or a subtile spec violation on our side.

The current plan is:

- [ ] officially support setting multisample count from outside. this is done in the branch mentioned above and will be merged for the next release
- [ ] mention problem in 'troubleshooting' page which can be included in manual and on github
- [ ] In the meanwhile we are trying to find out if driver bug or aardvark bug ;)

**Comment by haraldsteinlechner** [comment]
@hyazinthh do you know the status of this in the base libraries? should be fixed right? if so we could remove the workaround https://github.com/pro3d-space/PRo3D/blob/b145dfce1a2043c857b13d33e1d72b13b4ad7abd/src/PRo3D.Viewer/Program.fs#L201

**Comment by hyazinthh** [comment]
I don't think so: https://github.com/aardvark-platform/aardvark.rendering/issues/86

**Comment by haraldsteinlechner** [comment]
don't think so means - no not fixed, keep workaround, right?

**Comment by hyazinthh** [comment]
Yes, that's what your final comment on that issue suggests at least.
